app_dart/lib/src/foundation/utils.dart

```diff
@@ -103,6 +103,39 @@ Future<List<Map<String, dynamic>>> getRepoBuilders(HttpClientProvider branchHttp
       .toList();
 }
 
+/// Returns a LUCI [builder] list that covers changed [files].
+///
+/// [builders]: enabled luci builders.
+/// [files]: changed files in corresponding PRs.
+///
+/// [builder] is with format:
+/// {
+///   "name":"yyy",
+///   "repo":"flutter",
+///   "taskName":"zzz",
+///   "enabled":true,
+///   "run_if":["a/b/", "c/d/**"]
```

Comment:
What about single files?

Reply:
The case of single files has been covered. Please refer to the test below. Updated the doc to include a case of a single file and a case of single *